source/type/pt_data.cpp

```diff
@@ -35,6 +35,98 @@ www.navitia.io
 
 namespace navitia { namespace type {
 
+type::Network* PT_Data::get_or_create_network(const std::string& uri, const std::string& name, int sort) {
+    const auto it = networks_map.find(uri);
+    if (it != networks_map.end()) {
+        return it->second;
+    }
+
+    nt::Network* network = new nt::Network();
+    network->uri = uri;
+    network->name = name;
+    network->sort = sort;
```

Comment:
When you create a **Network** you need to fill
- std::vector<Line*> line_list;
- std::set<Dataset*> dataset_list; 
Because if we access into the code somewhere, this is a nullptr

Reply:
you probably create a fake **Dataset** struct for that and add the **Line** inside the line_list

Reply:
it is just to be good with respect to Data Model

Reply:
It is just to respect the Data Model

Reply:
okay, for `std::vector<Line*> line_list;` here
https://github.com/pbougue/navitia/pull/2/files#diff-12123f76f97a0b21544ff33ec37d8ffaR93